---
layout: page
title: Ongoing Projects
---

## Overview
- [Precision Medicine and Therapeutic Discovery](#premed)
- [Predict Chemical States in Random Heterogenous Polymers](#chemstate)
- [Bivariate Dependence Modeling and Biological Network Inference](#network)

---

## Precision Medicine and Therapeutic Discovery <a name="premed"></a>
> <img align="left" src="../img/sensitive.gif" width="250"> Rapidly decreasing costs of molecular measurement technologies not only enable profiling of disease sample molecular features (e.g., transcriptome, proteome, metabolome) but also enable measuring of molecular signatures of individual drugs in clinically relevant models. We are interested in exploring new computational and statistical methods to relate diseases to potentially efficacious drugs through various molecular features.

**People**: Calvin Chi, [Z. Tom Hu](http://tomhu.me), Zoe Vernon, Yuting Ye

**Collaborators**: [Professor Bin Chen's Group](http://binchenlab.org)

**Selected Publications**:

+ [AICM: A Genuine Framework for Correcting Inconsistency Between Large Pharmacogenomics Datasets](http://psb.stanford.edu/psb-online/proceedings/psb19/hu.pdf), Zhiyue Tom Hu, Yuting Ye, Patrick Newbury, Haiyan Huang and Bin Chen. In Proceedings of *Pacific Symposium on Biocomputing 2019*

+ Reverse Correlation for Translating Genomics Features into Therapeutics *(In Preperation)*

---

## Predict Chemical States in Random Heterogenous Polymers <a name="chemstate"></a>
> <img align="left" src="../img/HMM.png" width="250"> In this project, we are applying a Hidden Markov Model to statistically understand and predict the transmembrane segments in random heteropolymers (RHP) chains. We are now extending our Hidden Markov Model for large-scale hydrophobic states predictions by adapting variational inference and deep learning.

**People**: Yun Zhou

**Collaborators**: [Professor Ting Xu's Group](https://xugroup.berkeley.edu)

**Selected Publications**:

+ [Single-Chain Heteropolymers Transport Protons Selectively and Rapidly](https://www.nature.com/articles/s41586-019-1881-0), Tao Jiang, Aaron Hall, Marco Eres, Zahra Hemmatian, Baofu Qiao, Yun Zhou, Zhiyuan Ruan, Andrew D. Couse, William T. Heller, Haiyan Huang, Monica Olvera de la Cruz, Marco Rolandi, Ting Xu. *Nature 2020 Jan; 577(7789):216-20.*

+ Predicting Hydrophobic Segments by High Dimensional Inference and Machine Learning Approaches *(In Preperation)*

---

## Bivariate Dependence Modeling and Biological Network Inference <a name="network"></a>
> <img align="left" src="../img/network.png" width="250" hspace="20"> Networks pervade many disciplines of science for analyzing complex systems with interacting components. In particular, this concept is commonly used to model interactions between genes and identify closely associated genes forming functional modules to help obtain a more complete portrayal of gene dynamics. Research along this line has been of our interest for a long time.

**People**: Soyeon Ahn, Zoe Vernon, Yuting Ye, Yun Zhou

**Collaborators**: [Professor Peter Bickel](https://www.stat.berkeley.edu/users/bickel/), [Professor. Marisa Wong Medina](http://www.chori.org/Principal_Investigators/Medina_Marisa/medina_overview.html), [Professor Noam Shomoron](https://nshomron.github.io), [Professor. YX Rachel Wang](https://www.maths.usyd.edu.au/ut/people?sms=y&who=RYX_Wang), [Professor Michael S Waterman](https://dornsife.usc.edu/labs/msw)

**Selected Publications**:

+ [GeneFishing to Reconstruct Context Specific Portraits of Biological Processes,](https://www.pnas.org/content/116/38/18943) Ke Liu, Elizabeth Theusch, Yun Zhou, Tal Ashuach, Andrea C. Dose, Peter J. Bickel, Marisa W. Medina, Haiyan Huang. *PNAS September 17, 2019 116 (38) 18943-18950*

+ [Single Cell Clustering Based on Cell-Pair Differentiability Correlation and Variance Analysis,](https://www.ncbi.nlm.nih.gov/pubmed/29771290) Hao Jiang, Lydia L Song, Haiyan Huang, Luonan Chen. *Bioinformatics. 2018 May 16;1:11*

+ [Biclustering by Sparse Canonical Correlation Analysis,](https://link.springer.com/article/10.1007/s40484-017-0127-0) Harold Pimental, Zhiyue Tom Hu, Haiyan Huang. *Quantitative Biology. 2018 Mar 1;6(1):56-67*

+ [Generalized correlation measure using count statistics for gene expression data with ordered samples,](https://www.ncbi.nlm.nih.gov/pubmed/29040382) YX Rachel Wang, Ke Liu, Elizabeth Theusch, Jerome I Rotter, Marisa W Medina, Michael S Waterman, Haiyan Huang. *Bioinformatics. 2017 Oct 12;34(4):617-24.*

+ [SIDEseq: A Cell Similarity Measure Defined by Shared Identified Differentially Expressed Genes for Single-cell RNA Sequencing data,](https://www.researchgate.net/publication/317646092_SIDEseq_A_Cell_Similarity_Measure_Defined_by_Shared_Identified_Differentially_Expressed_Genes_for_Single-Cell_RNA_sequencing_Data) Courtney Schiffman, Christina Lin, Funan Shi, Luonan Chen, Lydia L Sohn, Haiyan Huang. *Statistics in Biosciences. June 2017, Volume 9, Issue 1, pp 200â€“216.*

+ [Inferring Gene-Gene Interactions and Functional Modules Using Sparse Canonical Correlation Analysis,](https://arxiv.org/pdf/1401.6504.pdf) YX Rachel Wang, Keni Jiang, Lewis J Feldman, Peter J. Bickel, Haiyan Huang. *Annals of Applied Statistics. 9(1): 300-323.*

+ [Gene Coexpression Measures in Large Heterogeneous Samples Using Count Statistics,](https://pdfs.semanticscholar.org/5c2e/528a5b9b84f91663a7590782097c29c5ae83.pdf) YX Rachel Wang, Michael S. Waterman, Haiyan Huang. *Proc Natl Acad Sci. USA. (PNAS) 111(46):16371-6*

+ [A statistical framework to infer functional gene associations from multiple biologically interrelated microarray experiments,](https://www.tandfonline.com/doi/abs/10.1198/jasa.2009.0037) Siew Leng Teng, Haiyan Huang. *Journal of the American Statistical Association*
